lago/sdk_utils.py

```diff
@@ -0,0 +1,74 @@
+import functools
+import inspect
+import wrapt
+
+
+class SDKWrapper(wrapt.ObjectProxy):
+    """A proxy object that exposes only methods which were decorated with
+    :func:`expose` decorator."""
+
+    def __getattr__(self, name):
+        attr = getattr(self.__wrapped__, name)
+        return filter_sdk(attr, name)
```

Comment:
I think that the name filter is misleading, I would go with ```check_if_exposed``` or ```validate_exposed``` or something similar.